### DISCUSSION TITLE
Types Filter as a checkbox instead of a single select

### DISCUSSION THREAD

**Discussion by steve970** [code4recovery/12-step-meeting-list#1537] [Q&A] [answered]
I am currently using TSML to create a list of international meetings. I've been playing with making the Types filter a dropdown box with checkboxes so that I can search for different types of meetings at once. For example. Let's say I filtered for all the meetings in California, I'd like to be able to search by Open OR LGBTQ meetings and have the filter return all the open meetings in California OR all the LGBTQ in California. I've created the dropdown and the logic for how it works UX-wise. I am looking for some guidance about sending an Array of Types to query results. Thanks! Please let me know if I can provide any additional information.

**Comment by joshreisner** [comment, marked as answer]
hi @steve970 are you using TSML UI or Legacy? since you're asking about sending an array i'm guessing you're using Legacy? one reason to consider TSML UI is that it has a timezone-agnostic option

it's a little-known fact that we do actually support multiple selections in the types menu on both of these, but it's hard to discover and limited to desktop only. we have a [backlog ticket to update the filters UI for this](https://github.com/code4recovery/tsml-ui/issues/171) on TSML UI but the limitation for us has been not finding a good design, so i'm interested to see what you come up with

the way you trigger this mode is to hold down the command key on mac, or the equivalent key on windows (maybe it's the windows key or the ctrl key?) while you make your second click. 

caveats: 
* this only works on the types menu in legacy, using AND logic (not OR logic)
* on TSML UI it applies OR logic on days, times, and regions, but AND logic on types

anyway, to answer your question, you can concat the `type` with a comma in your POST in legacy

<img width="1728" alt="Screenshot 2024-10-09 at 3 08 06 PM" src="https://github.com/user-attachments/assets/c1e15b7d-b2a3-4222-9e81-e764f9c2e1f6">
